sdk/search/azure-search-documents/src/main/java/com/azure/search/annotation/SearchableFieldProperty.java

```diff
@@ -0,0 +1,25 @@
+package com.azure.search.annotation;
```

Comment:
Curious why this doesn't have default values like the FieldProperty?

Reply:
Why doesn't this extend from the one above?

Reply:
Looking at the properties on FieldProperty, couldn't we just use "isSearchable" to be true? I don't see the extra value in adding this annotation.

Reply:
The namespace should go for com.azure.search.documents.indexes.annotation.

Reply:
I am worrying customer has trouble to find this. I will move to com.azure.search.documents.indexes first. Will check with UX study